src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -532,9 +534,11 @@ private void updateParticipantMetadata(
         // error means there is no metadata yet.
         Map<FullQualifiedName, Set<Object>> entity = new HashMap<>();
         try {
-            entity = dataApi
-                    .getEntity( entitySetIdMap.get( METADATA_ENTITY_SET_NAME ), metadataEntityKeyId );
-        } catch ( RhizomeRetrofitCallException e ) {}
+            entity = dataApi.getEntity( entitySetIdMap.get( METADATA_ENTITY_SET_NAME ), metadataEntityKeyId );
+        }
+        catch ( Exception e ) {
+            logger.error( "study = {} participant = {}", studyId, participantId, e );
```

Comment:
Might be helpful to include a description of the context of the error, like "Unable to load metadata entity: study = {} participant = {}" or something like that

Reply:
I'll update. I was thinking the stacktrace has plenty of context.